subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiExecution.groovy

```diff
@@ -38,7 +38,7 @@ class ToolingApiExecution extends AbstractMultiTestRunner.Execution implements T
 
     @Override
     protected String getDisplayName() {
-        return "${displayName(toolingApi.version)} -> ${displayName(gradle.version)}"
+        return "TAPI: ${displayName(toolingApi.version)}, target: ${displayName(gradle.version)}"
```

Comment:
I am not sure this makes things easier to understand. Maybe `TAPI` and `Gradle` would be better. Or something like `TAPI 6.7 -> Gradle 4.5`. Do you think it makes sense to get some feedback on this rename on Slack?

Reply:
@donat what do you think?

Reply:
I'm fine even with the original notation, but I guess I just got used to it. 

The `TAPI 6.7 -> Gradle 4.5` notation is a bit more explicit, I like it better than `target`.

Reply:
`TAPI 6.7 -> Gradle 4.5` it is!